cmd/j5/main.go

```diff
@@ -262,6 +262,7 @@ func runProtoBuild(ctx context.Context, cfg struct {
 	} else if cfg.CommitHash == "" || cfg.CommitTime == "" {
 		return fmt.Errorf("commit hash and time are required, or set --git-auto")
 	} else {
+		commitInfo = &builder_j5pb.CommitInfo{}
```

Comment:
It won't let me place the suggestion correctly, but I would encourage we go for setting the values in the struct instantiation. We've found in the past that people will mistakenly set a field twice a few lines apart and the compiler doesn't complain about that. Where as if you do it in the struct instantiation it will complain if you set a field twice.

```suggestion
		commitTime, err := time.Parse(time.RFC3339, cfg.CommitTime)
		if err != nil {
			return fmt.Errorf("parsing commit time: %w", err)
		}
		
		commitInfo = &builder_j5pb.CommitInfo{
			Hash: cfg.CommitHash,
			Time: commitTime,
			Aliases: cfg.CommitAliases,
		}
```